sonar-core/src/test/java/org/sonar/core/dashboard/ActiveDashboardDaoTest.java

```diff
@@ -23,8 +23,7 @@
 import org.junit.Test;
 import org.sonar.core.persistence.AbstractDaoTestCase;
 
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class ActiveDashboardDaoTest extends AbstractDaoTestCase {
```

Comment:
AbstractDaoTestCase should not be used anymore (I will deprecate it in order to not forget to remove it each time we see it!), it should be replaced by a call to DbTester.